contracts/KetlAttestation.sol

```diff
@@ -81,7 +81,8 @@ contract KetlAttestation is ERC1155, Ownable, Versioned, ERC2771Recipient {
   mapping(uint => IncrementalTreeData) public entanglementsTrees;
   mapping(uint => uint[]) public entanglements;
   mapping(uint => mapping(uint => bool)) public entanglementsRoots;
-  mapping(uint => bool) public attestationHashesEntangled;
+  mapping(uint => Counters.Counter) public attestationHashesEntangledCount;
+  uint public maximumEntanglementsPerAttestation = 1;
```

Comment:
Got it. We would need to handle the default case though since every value in the map would be 0 by default – so I'd like to keep this maybe as `globalMaximumEntanglementsPerAttestation`; and then we can have a separate `maximumEntanglementsPerAttestation` map.

If `maximumEntanglementsPerAttestation[attestationHash]` returns `0`, then we default to `globalMaximumEntanglementsPerAttestation`. If not, then we use the value. 

What do you think?

Reply:
Updated.